spyder/utils/encoding.py

```diff
@@ -111,6 +113,17 @@ def get_coding(text):
             # sometimes we find a false encoding that can result in errors
             if codec in CODECS:
                 return codec
+
+    # Falback using chardet
+    if is_binary_string(text):

```

Comment:
@ccordoba12 I think that  It's better to relay in the information of the file that could be more accurate, and use encoding detection as a fallback, chardet recommend [that way](http://chardet.readthedocs.io/en/latest/faq.html#yippie-screw-the-standards-ill-just-auto-detect-everything) (although it make reference to mime types)

@goanpeca yep, duck-typing I'll correct it :)